### ISSUE TITLE
If HeaderFilter only for frozen columns then input-fields not shown

### ISSUE THREAD

**Issue by Rodo1982** [olifolkerd/tabulator#1232]
If you freeze the first columns and apply only headerfilters for the freezed columns, then the input-fields are not shown. In my example I also added topCalculation on the non-freezed columns. E.g., in the screenshot the first three columns are freezed plus headFilter="input", but you see only grey lines below the header title:

![screenshot](https://user-images.githubusercontent.com/39951139/42060391-e8b225ae-7b26-11e8-82be-9f3677066e61.jpg)

**Comment by olifolkerd** [comment]
Hey @Rodo1982 

Sorry to hear you are having issues,

Could you post a copy of your table constructor object so i can dun some tests against a similar setup?

Cheers

Oli :)

**Comment by Rodo1982** [comment]
Hi Oli,

here you are:

```javascript
$("#example-table2").ready(function() {
$("#example-table2").tabulator({
data:tabledata,
layout:"hide", //fit columns to width of table (optional)
paginationSize:20,
pagination:"local",
  
columns:[ //Define Table Columns
  
 { "field":"rscSegment", title:"Route", frozen:true, headerFilter:"input", },
 { "field":"pax", title:"Pax", align:"right", width:"auto", formatter:"money", formatterParams:{precision:0,symbol:"",symbolAfter:false}, topCalc:"avg", topCalcFormatter:"money", topCalcFormatterParams:{precision:0,symbol:"",symbolAfter:false}, bottomCalc:"sum", bottomCalcFormatter:"money", bottomCalcFormatterParams:{precision:0,symbol:"",symbolAfter:false}, },
 ],
tooltips:function(cell){
return  cell.getColumn().getField() + ":" + cell.getValue(); //return cells "field:value";
 },
 });
  
 var tabledata = [
  
 { "id":"1", "rscSegment":"A", "pax":12282.0},
 { "id":"2", "rscSegment":"B", "pax":164302.0},
 { "id":"3", "rscSegment":"C", "pax":33219.0},
  ];
   
  $("#example-table2").tabulator("setData", tabledata);
   
  });
```


Thanks in advance.
Robert :)

**Comment by olifolkerd** [comment]
Hey @Rodo1982 

Sorry for the delay on getting back to you on this, its been a bit manic getting ready for the v4.0 launch,

that is definitely a bug, i will make sure it is fixed in the 4.0 release.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
I have pushed a fix for this to the 4.0 branch and will include it in the next release.

Cheers

Oli :)
